### PR TITLE
Rewrite library: finalize rewrite (4/4)

### LINKED ISSUE #12: Use primary text color for library name
looks nicer?

### PR DESCRIPTION
**Pull request by SUPERCILEX**
3/4 fixed #12

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
Alrighty, this PR is ready for review! 😄

File: README.md [hunk 1]
```diff
@@ -1,94 +1,92 @@
-LicenseAdapter
-===
+# License Adapter
```
Comment (yshrsmz) [inline README.md]: I prefere LicenseAdapter

File: README.md [hunk 2]
```diff
@@ -1,94 +1,92 @@
...
 
 [![Build Status](https://travis-ci.org/yshrsmz/LicenseAdapter.svg?branch=master)](https://travis-ci.org/yshrsmz/LicenseAdapter)
 [![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-LicenseAdapter-green.svg?style=true)](https://android-arsenal.com/details/1/3516)
 [![Bintray](https://img.shields.io/bintray/v/yshrsmz/maven/licenseadapter.svg)](https://bintray.com/yshrsmz/maven/licenseadapter/view)
 
-Adapter library for RecyclerView to display your app's OSS dependencies.
+License adapter is an easy-to-use library that provides a RecyclerView
```
Comment (yshrsmz) [inline README.md]: LicenseAdapter

File: CHANGELOG.md [hunk 3]
```diff
@@ -31,7 +74,7 @@ Thanks [@SUPERCILEX](https://github.com/SUPERCILEX) for adding great new feature
 
 ### Enhancement
 
-- Add `Licenses#fromGitHubApacheV2(String)`, `Licenses#fromGitHubApacheV2(String, String)`, `Licenses#fromGitHubMIT(String)`, `Licenses#fromGitHubMIT(String, String)`, `Licenses#fromGitHubBSD(String)`, `Licenses#fromGitHubBSD(String, String)`.
```
Comment (yshrsmz) [inline CHANGELOG.md]: this 2 white spaces at the end of line should be preserved(for new line)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,46 @@
+## 2.0.0 - 2017/12/XX
+
+### Breaking changes
```
Comment (yshrsmz) [inline CHANGELOG.md]: awesome 👍

File: README.md (hunk 1 above)
Comment (SUPERCILEX) [reply README.md]: SGTM, I thought it was a typo or something. 😄

File: CHANGELOG.md (hunk 3 above)
Comment (SUPERCILEX) [reply CHANGELOG.md]: Oh, that's a cool trick! IntelliJ keeps trying to autoformat those spaces away so feel free to ping me if this change gets added again. 😄

**Comment by SUPERCILEX** [comment]
@yshrsmz For the notifications comment, that was the best I could do. The "demo mode" developer option still shows some status icons even though they're all fake—I definitely wasn't coding at 8 😉. (Also, I have one less notification than the current demo gif so I beat you. 😆)

To address your second point, I actually removed the url on purpose. Here's my reasoning:
- The `colorAccent` text is supposed to signify that a link is present.
- Displaying the url feels like a developer centric view since most users won't know what GitHub is or even what those library names mean.
- For developers, since GitHub is the only library that has a url anyway, they can know the url just though the `author/libName` syntax.
- And just for the sake of completion: adding a url made the UI feel bloated and was distracting from the actual library name (which is what's important IMO).

@yshrsmz What do you think of the url situation given the above arguments?

**Comment by yshrsmz** [comment]
>The "demo mode" developer option still shows some status icons

What I meant is not status/notification icons, but notifications itself(I can see hangout and some other notifications in the first few frames). But if it's hard to remove those notifications, then it's fine :)

and about the second point, I can completely agree with your opinion! Let's leave it as it is. thanks!

**Comment by SUPERCILEX** [comment]
@yshrsmz Ohhhhhhhh, I just rewatched the gif and saw what you meant. Thanks! Yeah, I don't want people to see my notifications either 😄. That should be fixed now.

**Comment by yshrsmz** [comment]
Ok, thanks for your great contribution!
will do the final check & release within a (hopefully) few days!

btw, do you mind if I add you as a collaborator of this repo?

**Comment by SUPERCILEX** [comment]
Awesome, thanks for working with me on this! I'll be on the lookout for that release. 😀 And yeah, I'd be happy to be added as a contributor. 😊

**Comment by yshrsmz** [comment]
just released 2.0.0! 
Thanks again for your contribution 👍

**Pull request merged**